camera/src/main.py

```diff
@@ -33,8 +39,7 @@ def ctrl_c_handler(sig, frame):
 signal.signal(signal.SIGINT, ctrl_c_handler)
 
 while running:
-    char = sys.stdin.read(1)
-    if char == 'q':
-        running = False
+    sleep(1)
+    running = pipeline.check_alive() and heartbeat.is_connected()
```

Comment:
If the detection process stops because the last heartbeat was too long ago, will it respawn eventually?

Reply:
There will be another process I think monitoring it. Likely the one that communicates with the phone for wifi/token and also deals with auth in general. Not quite sure how I want to handle it yet.